tasks/release.sh

```diff
@@ -39,23 +39,6 @@ if [ -n "$(git status --porcelain)" ]; then
   exit 1;
 fi
 
-# Update deps
-rm -rf node_modules
-rm -rf ~/.npm
-npm cache clear
-npm install
-
-cd packages/react-scripts
-# Force dedupe
-npm dedupe
-
-# Don't bundle fsevents because it is optional and OS X-only
-# Since it's in optionalDependencies, it will attempt install outside bundle
-rm -rf node_modules/fsevents
-
-# This modifies package.json to copy all dependencies to bundledDependencies
-node ./node_modules/.bin/bundle-deps
-
```

Comment:
Maybe it was only true for earlier versions with a different dependency structure or something like this.